UPGRADE.md

```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work

```

Comment:
might be too brief and misses a link to the docs

Reply:
I would suggest: 
- for [product search via Elasticsearch](/docs/introduction/product-search-via-elasticsearch.md) to work properly, you'll have to:

`- for [product search via Elasticsearch](/docs/introduction/product-search-via-elasticsearch.md) to work properly, you'll have to:`